### ISSUE TITLE
Failing tests in latest release

### ISSUE THREAD

**Issue by MementoRC** [polkascan/py-scale-codec#121]
Hi, the conda build fails for the following tests:
```
FAILED test/test_create_extrinsics.py::CreateExtrinsicsTestCase::test_create_extrinsic_bytes_signature - ValueError: Element "mode" of struct is missing in given value
FAILED test/test_create_extrinsics.py::CreateExtrinsicsTestCase::test_create_extrinsic_metadata_v14 - ValueError: Element "mode" of struct is missing in given value
FAILED test/test_create_extrinsics.py::CreateExtrinsicsTestCase::test_create_mortal_extrinsic - ValueError: Element "mode" of struct is missing in given value
FAILED test/test_create_extrinsics.py::CreateExtrinsicsTestCase::test_create_multisig_extrinsic - ValueError: Element "mode" of struct is missing in given value
FAILED test/test_create_extrinsics.py::CreateExtrinsicsTestCase::test_payment_info - ValueError: Element "mode" of struct is missing in given value
```

We could remove running `pytest -v test` post-build, or add a patch that remove these tests from the suite (we only want to run as many upstream tests as possible, just as a peace of mind for conda builds quality)

Let me know what's you preferred approach
Thanks

**Comment by arjanz** [comment]
This is related to a recent release of https://github.com/polkascan/py-substrate-interface/releases/tag/v1.7.9

This release goes hand-in-hand with https://github.com/polkascan/py-scale-codec/releases/tag/v1.2.10 (package requirement was also updated). I wonder why the conda build would fail because of this?

**Comment by MementoRC** [comment]
Right. Do you foresee that this would be a one-time issue or more likely to occur down the line. I was thinking to deactivate the test for this release then reactivate them once the egg is hatched

**Comment by MementoRC** [comment]
Resolved.

**Comment by arjanz** [comment]
> Right. Do you foresee that this would be a one-time issue or more likely to occur down the line. I was thinking to deactivate the test for this release then reactivate them once the egg is hatched

This occurred before and will occur when extrinsic related changes are made, this requires changes in the SCALE type definition and usage of this type in py-substrate-interface. I think it's better to keep the dependency one way,

**Comment by MementoRC** [comment]
Ok, I actually initially didn't get it right,I thought it was a testsuite discrepancy, but it is the downstream link that creates the lock. I'll remove it when I reinstate the tests. Thanks
